### PR TITLE
Doc: Add docs for extending integrations with filter-elastic_integrations

### PR DESCRIPTION
**Pull request by karenzone**
Fixes:  #15172
Replacement for #15225

**PREVIEW:** https://logstash_15518.docs-preview.app.elstc.co/guide/en/logstash/master/ea-integrations.html

### CODE REVIEW

File: docs/static/ea-integrations.asciidoc [hunk 1]
```diff
@@ -0,0 +1,40 @@
+[[ea-integrations]]
+== Using {ls} with Elastic {integrations} (Beta)
```
Comment (karenzone) [inline docs/static/ea-integrations.asciidoc:2]: ToDo: Research proper tagging and messaging for Beta features

**Comment by karenzone** [comment]
@flexitrev, I've reworked this section based on your comments. 
 I'd like to take a more incremental, iterative approach to merging content.  Ideally, I'd like to get this overview section merged while I develop the config samples.  

Please LMKWYT! Thanks for your help with this.

**Comment by karenzone** [comment]
@flexitrev, I'm glad that you've already found this content helpful. Thanks for sharing the feedback. ❤️ 
I've added a config sample. Please LMKWYT.

**Comment by karenzone** [comment]
MERGE PENDING plugin repo becoming publicly available.
In the meantime, we're still accepting review comments and iterating.

**Review by flexitrev — APPROVED** [review]
Docs looks great. Should be good for getting users started.

**Comment by karenzone** [comment]
#### Update
`filter-elastic_integration` repo is now publicly available.  Merge pending another merge to update docs tooling, and initial docs generation for new docs.

**Comment by karenzone** [comment]
@elasticmachine run elasticsearch-ci/docs

File: docs/static/ea-integrations.asciidoc [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+[discrete]
+[[integrations-and-ls]]
+=== Extend {integrations} with {ls} (Beta)
+
+Logstash can run the ingest pipeline component of your integration when you use the Logstash filter-elastic_integration plugin. 
+
+.How to
+
+****
+Create a {ls} pipeline that uses the <<plugins-inputs-elastic_agent,elastic_agent input>> plugin, and the https://github.com/elastic/logstash-filter-elastic_integration[logstash-filter-elastic_integration] plugin as the _first_ filter in your {ls} pipeline.
```
Comment (karenzone) [inline docs/static/ea-integrations.asciidoc:33]: Note to self @karenzone: Update with links to plugin docs after they are generated

**Comment by karenzone** [comment]
49793

**Comment by karenzone** [comment]
@logstashmachine backport 8.11

**Comment by mbudge** [comment]
Where’s the documentation for this feature? We want to try it out.

**Comment by flexitrev** [comment]
https://www.elastic.co/guide/en/logstash/current/ea-integrations.html

Please give us feedback on anything else you'd like to see.

**Pull request merged**